src/oInput/oInput.js

```diff
@@ -0,0 +1,98 @@
+import mapEvents from '../utils/eventMapper';
+import './input.css';
+import o from 'ojs-core';
+
+const TYPES = {
+    text: 'Text field',
+    date: 'Date field',
+    email: 'E-mail field',
+    month: 'Month field',
+    number: 'Number field',
+    password: 'Password field',
+    tel: 'Tel field',
+    file: 'File field',
+    url: 'URL field',
+};
+
+class oInput {
+    constructor(config) {
+        this.store = {
+            label: '',
+            type: 'text',
```

Comment:
type: text, a typeText: false. Czym jest typeText w takim razie?

Reply:
typeText może faktycznie niefortunna nazwa. To jest text, którym możesz opisać sobie jakiego typu jest to pole. Tzn ten napisz po prawej stronie nad inputem możesz sobie zmodyfikować

Reply:
Nie podoba mi się do końca ten zamysł. Bo jeśli dobrze rozumiem, to teraz jak chcesz sobie zrobić na przykład, że input type text ma napis "pole tekst" to w każdym inpucie przy tworzeniu trzeba podać ten text? Może jakieś pole/config statyczny dla konkretnych typów, które będą wartościami domyślnymi (wartości domyślne i tak masz, ale żeby można było je zastąpić swoimi domyślnymi).